appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js

```diff
@@ -34,6 +34,60 @@ describe('manage docs on appellant case', () => {
 		});
 	});
 
+	it('upload doc and remove when final version', () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.uploadDocAppellantCase(caseRef);
+			caseDetailsPage.clickAddAdditionalDocs();
+			caseDetailsPage.uploadSampleFile(sampleFiles.document);
+			caseDetailsPage.checkFileNameDisplays('sample-file.doc');
```

Comment:
Same point as Rachel made below

Reply:
Updated to reflect Rachel's comments above